net-snmp/run.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+(
+    . /etc/sysconfig/snmpd
+    if test "$WAIT_FOR_CONFIGURATION" == 1; then
+        systemd-notify --ready
+        exec sleep 9999d
```

Comment:
@giuseppe what about having ``run.sh`` check to see if the user is created. If it doesn't exist, it creates and runs the daemons. If it does exist, it goes directly to starting the daemons? The creation of the user can be looked at with something like ```egrep 'r[wo]?user' /etc/snmp/snmpd.conf EXPECTED_USER```.